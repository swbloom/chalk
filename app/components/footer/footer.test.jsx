import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index.jsx';

test('It renders the footer component', () => {
    const component = renderer.create(
        <Footer />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});