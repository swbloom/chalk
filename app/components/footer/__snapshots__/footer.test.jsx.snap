// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders the footer component 1`] = `
<footer
  className="mainFooter card"
>
  <h3>
    HackerYou
  </h3>
  <ul>
    <li>
      <a
        href="http://hackeryou.com"
      >
        <i
          className="chalk-link"
        />
        hackeryou.com
      </a>
    </li>
    <li>
      <a
        href="mailto:info@hackeryou.com"
      >
        <i
          className="chalk-email"
        />
        info@hackeryou.com
      </a>
    </li>
    <li>
      <a
        href="http://github.com/hackeryou"
      >
        <i
          className="chalk-github"
        />
        github.com/hackeryou
      </a>
    </li>
    <li>
      <a
        href="http://twitter.com/hackeryou"
      >
        <i
          className="chalk-twitter"
        />
        @hackeryou
      </a>
    </li>
  </ul>
  <p>
    <strong>
      Copyright 
      2017
       
      <a
        className="blackLight"
        href="http://hackeryou.com"
      >
        HackerYou
      </a>
    </strong>
    <br />
    <small>
      The contents of this site are the property of HackerYou. No portion of this site is to be shared without permission.
    </small>
  </p>
</footer>
`;
